Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -50,6 +50,8 @@ enum PurchaseStrings: CustomStringConvertible {
     static let purchases_delegate_set_to_nil = "Purchases delegate is being set to nil, " +
         "you probably don't want to do this."
     static let management_url_nil_opening_default = "managementURL is nil, opening Apple's subscription management page"
+    static let unable_to_get_intro_eligibility_for_user = "Unable to get intro eligibility for appUserID: %@"
+    static let unable_to_get_intro_eligibility_with_error = "Unable to get intro eligibility: %@"
```

Comment:
we should make these `case`, but it'll require a rebase from `main` to minimize the number of conflicts. 

I rebased `storekit_2` from `main` just now locally, and it did take some work, but it's ready. @vegaro would you like me to hold until this PR gets pushed in order to force push the rebased `storekit_2`, or would you like me to push it now so that you can rebase this PR on that?